packages/devextreme/js/__internal/grids/tree_list/rows/m_rows.ts

```diff
@@ -121,12 +120,16 @@ class TreeListRowsView extends RowsView {
   protected _createRow(row) {
     const node = row && row.node;
     const $rowElement = super._createRow.apply(this, arguments as any);
-
     if (node) {
       this.setAria('level', row.level + 1, $rowElement);
-
       if (node.hasChildren) {
-        this.setAria('expanded', row.isExpanded, $rowElement);
+        const { cells } = row;
```

Comment:
Why do you need this block of changes?

Reply:
Line 126 - 131: This adds aria-expanded attribute to <td> elements on its creation. This is also the line of codes being called on expand/collapse toggle. Maybe there was a cache on your end which causes the changes to not apply.

Line 132: This adds the aria-expanded attribute to <tr> element on row creation.. same as above, this is being called on expand/collapse toggle.